api/app/reviews/models.py

```diff
@@ -100,4 +100,16 @@ def __init__(self,
                  review_question_id,
                  value):
         self.review_question_id = review_question_id
-        self.value = value
\ No newline at end of file
+        self.value = value
+
+
+class ReviewConfiguration(db.Model):
+    id = db.Column(db.Integer(), primary_key=True)
+    review_form_id = db.Column(db.Integer(), db.ForeignKey('review_form.id'), nullable=False)
+    num_reviews_required = db.Column(db.Integer())
+    num_optional_reviews = db.Column(db.Integer())

```

Comment:
I think specify `nullable=False` here as well and then do another migration (having 2 will be fine)

Reply:
I think the default is `nullable=False`. Happy to make this explicit though.

Reply:
Particularly since I did it on all the other fields 😓

Reply:
Na if you look at the migration, they were set to be nullable.
Unless do you know of a way to configure SQL Alchemy to make fields not nullable by default?

Reply:
You're completely right.